### ISSUE TITLE
Cart

### ISSUE THREAD

**Issue by Nethmee** [bitsoferised/ShareABook_Frontend_v.2#64]
My cart has been cleared once I logged out. I suggest giving some time period just like Dinesh sir suggested.

**Comment by dilinipeirs** [comment]
to do this the data will have to be stored in the database when logged out. ill try to create an endpoint for that

**Comment by RashmiChrystal** [comment]
Guys, users can add products from the product view page to the cart. It was fixed and was committed to the branch named issues.

**Comment by dilinipeirs** [comment]
@RashmiChrystal i think its good u put pull requests assigned @Nethmi96 as a reviewer. then she can merge your work to the frontend master too ne..

**Comment by Nethmi96** [comment]
> to keep the data even after logging out, the data will have to be stored in the database when logged out.
> 
> but then again, our site's having products that can be easily sold out right @Nethmee ? Then is there a point in actually saving the items in the cart to the database once logged out? We can give a msg when logging out such as [the items in your cart will be removed] or something once trying to log out maybe?
> 
> @Nethmee @Nethmi96 @RashmiChrystal what are your ideas?
> 
> cart is getting cleared once logged out because the frontend storages are cleared upon logging out.

Yes, i think that solution is better. Because if we are not displaying products to other users' once it is added to the cart by an individual user, we cannot let them keep those materials in their cart forever

**Comment by Nethmi96** [comment]
> Guys, users can add products from the product view page to the cart. It was fixed and was committed to the branch named issues.

Rashmi did you push this? I can't find any commits to the branch issue after the one you committed by changing feedback tables into mat-tables
